src/styles/views/Welcome/view.module.sass

```diff
@@ -1,56 +1,107 @@
 @use "../../variables"
 @import url("https://fonts.googleapis.com/css2?family=Ubuntu:wght@700&display=swap")
-  
-.Welcome
-  position: relative
-  height: 100vh
-
-  .Card
-    position: absolute
-    top: 50%
-    left: 50%
-    text-align: center
-    width: 30% !important
-    transform: translate(-50%, -50%)
-
-    @media screen and (max-width: variables.$smaller-screens)
-      width: 53% !important
-
-    @media screen and (max-width: variables.$mobile-screens)
-      width: 88% !important
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@100;200;300;400;500;600;700;800&display=swap')
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@100;200;300;400;500;600;700;800&family=Roboto+Mono:wght@400;500;600&display=swap')
```

Comment:
We don't need the inter font, why do we have that here?
Also Poppins is already added through the ui lib